mlrun/serving/server.py

```diff
@@ -383,8 +383,15 @@ def v2_serving_handler(context, event, get_body=False):
         if event.body == b"":
             event.body = None
 
-    # ML-6065 – workaround for NUC-178
-    if hasattr(event, "trigger") and event.trigger.kind in ("kafka", "kafka-cluster"):
+    # ML-6065
```

Comment:
please remove reference to ML-6065 (as I would like to resolve this with ML-6863 at 1.7)